rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -92,6 +94,12 @@ local function format_ipv6_endpoints(endpoints)
   return formatted_endpoints
 end
 
+local function use_external_name(backend)
```

Comment:
Similar to https://github.com/kubernetes/ingress-nginx/pull/5418/files#r412948805, I'd go with `is_backend_with_external_name`.